app/src/main/java/com/capstone/weatherapp/City.kt

```diff
@@ -0,0 +1,5 @@
+package com.capstone.weatherapp
+
+class City (val name: String, val temp: Int) {
```

Comment:
This class actually isn't needed at all. You'll create a data class for the api data later on but this isn't needed for the recyclerView